airflow/www/views.py

```diff
@@ -4162,7 +4162,7 @@ def class_permission_name(self):
         """Returns appropriate permission name depending on request method name."""
         if request:
             action_name = request.view_args.get("name")
-            _, method_name = request.url_rule.endpoint.split(".")
+            _, method_name = request.url_rule.endpoint.rsplit(".", 1)
```

Comment:
Oops, this is unrelated and should have been submitted separately. A trivial cleanup though so probably OK to have it here.